app/models/expression_file_info.rb

```diff
@@ -47,14 +47,16 @@ class ExpressionFileInfo
                                 '10x scATAC-seq', # scATAC-seq
                                 '10x TCR enrichment', # targeted transcriptomic
                                 '10x Visium', # spatial transcriptomic
-                                'CEL-seq2', # scRNAseq
+                                '10x Xenium', # spatial transcriptomics
+				'CEL-seq2', # scRNAseq
                                 'Drop-ChIP', # scChIP-seq
                                 'Drop-seq', # scRNAseq
                                 'dsc-ATAC-seq', # scATAC-seq
                                 'dsci-ATAC-seq', # scATAC-seq
                                 'inDrop', # scRNAseq
                                 'MARS-seq', # scRNAseq
                                 'MERFISH', # spatial transcriptomic
+				'Nanostring CosMx', #spatial transcriptomic
```

Comment:
This canonical [casing](https://www.google.com/search?q=nanostring+cosmx) seems to "NanoString" rather than "Nanostring".

```suggestion
				'NanoString CosMx', #spatial transcriptomic
```

Reply:
Great catch! Done (27a98b4b1)